core/src/main/java/org/apache/iceberg/MetadataUpdateParser.java

```diff
@@ -126,6 +127,9 @@ private MetadataUpdateParser() {}
   // SetCurrentViewVersion
   private static final String VIEW_VERSION_ID = "view-version-id";
 
+  // RemovePartitionSpecs
+  private static final String PARTITION_SPEC_IDS = "partition-spec-ids";
```

Comment:
please add tests for this to `TestMetadataUpdateParser`

Reply:
also this is called `spec-ids` in the OpenAPI definition: https://github.com/apache/iceberg/blob/d368a5f84448b2e5698f9cab668a80eda6e3f96d/open-api/rest-catalog-open-api.yaml#L2958